app/models/invoice.rb

```diff
@@ -4,7 +4,7 @@ class Invoice < ApplicationRecord
   self.belongs_to_required_by_default = false
 
   belongs_to :order, class_name: 'Spree::Order'
-  serialize :data, Hash
+  serialize :data, Hash, coder: YAML
```

Comment:
From my local tests, I think we'll need to add `type:` before hash when we upgrade to 7.1.